eclair-core/src/test/scala/fr/acinq/eclair/EclairImplSpec.scala

```diff
@@ -155,6 +158,46 @@ class EclairImplSpec extends TestKit(ActorSystem("mySystem")) with fixture.FunSu
       }
     })
   }
+import scala.collection.mutable.ListBuffer
```

Comment:
Can you please order imports and run the formatter?
If you're using IntelliJ, `ctrl+alt+o` and `ctrl+alt+l`.